lib/WALASupport/SILWalaInstructionVisitor.cpp

```diff
@@ -300,7 +300,7 @@ jobject SILWalaInstructionVisitor::visitAllocExistentialBoxInst(AllocExistential
 
     auto name = "ExistentialBox:" + 
       AEBI->getFormalConcreteType().getString() + "->" + AEBI->getExistentialType().getAsString();
-    SymbolTable.insert(((char *)AEBI) + sizeof(SILInstruction) , name);
+    SymbolTable.insert(static_cast<ValueBase *>(AEBI), name);
```

Comment:
Aren't these completely different values? In the first one, we're also annotating the string with the size of the SILInstruction, while in the second, we're adding the ValueBase for AEBI.

Reply:
Annotating the string with the size of the SILInstruction was the change I made in my last pull request, which is essentially doing the same thing as the cast. I found it cleaner and easier to understand with a cast than annotating a pointer with some value.

Reply:
I didn't realize that a cast will do the work when I was making my last PR, and that's why I made the old one. Please consider this as a fix of my last PR for AEBI and PEBI.

Reply:
Ah, I see your point now.